src/lib/mongoBackend/mongoUpdateContextSubscription.cpp

```diff
@@ -231,6 +231,25 @@ HttpStatusCode mongoUpdateContextSubscription
     }
   }
 
+
+  //
+  // StringFilter in Scope?
+  //
+  Restriction    restriction;
+  StringFilter*  stringFilterP = NULL;
+
+  for (unsigned int ix = 0; ix < requestP->restriction.scopeVector.size(); ++ix)
```

Comment:
Same comment as in https://github.com/telefonicaid/fiware-orion/pull/1900/files#r57232596